### PR TITLE
Fix #7564: virtual-chassis position zero

### LINKED ISSUE #7564: virtual-chassis position zero
### NetBox version

v3.0.7

### Python version

3.7

### Steps to Reproduce

1. Create 2 devices
2. Create a new virtual chassis
3. Set the Initial position to zero
4. add the 2 devices
5. Create the Virtual Chassis

Next one, also related
1. Edit the virtual chassis
2. Set the posistions to 0 an 1
3. Save the VC
4. check the overview of devices in the VC

### Expected Behavior

A new VC is created with 2 members. with position 0 en 1.
The GUI shows the 0 and 1 position in a gray box

### Observed Behavior

The members are created with position 1 and 2.
image

After correcting: The position 0 is not displayed
image

### PR DESCRIPTION
**Pull request by PieterL75**
### Fixes: #7564 

When a new VC was created with a VC start position of zero, it was created with a start position of 1.
When a VC was displayed that had a position of zero, the label was not displayed in the VC overview.

### CODE REVIEW

**Review by jeremystretch — CHANGES_REQUESTED** [review]
Thanks for taking this on!

**Review by PieterL75 — COMMENTED** [review]
Ok

File: netbox/dcim/forms/object_create.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def save(self, *args, **kwargs):
 
         # Assign VC members
         if instance.pk:
-            initial_position = self.cleaned_data.get('initial_position') or 1
+            initial_position = self.cleaned_data.get('initial_position') if self.cleaned_data.get('initial_position') != None else 1
```
[the remark this answers is not in the document]
Comment (jeremystretch) [reply netbox/dcim/forms/object_create.py]: @PieterL75 this still needs to be addressed. What is the issue?

[the remark this answers is not in the document]
Comment (PieterL75) [reply netbox/dcim/forms/object_create.py]: Jeremy,
The proposed solution with .get did not work.
I submitted a new update with the "if self.cleaned_data.get('initial_position') != None else 1".
See below in the pull request

Pieter

**Comment by jeremystretch** [comment]
Ok, I dug into this a bit and I see why you were getting stuck. The VC creation is missing a piece of validation logic to ensure that `initial_position` is set if adding VC members.

Also, the badge display needed to be fixed by passing `show_empty=True`; casting the value to a string would break its usage elsewhere in the application.

I'm going to push a separate commit which includes the form validation fix just to get this knocked out, but thank you for tackling this! Sorry it took so long.

**Pull request closed without merging**